Dockerfile

```diff
@@ -0,0 +1,84 @@
+# build via (docker root = project root):
+# docker build -t absaoss/eventgate:latest .
+#
+# build [with customizations] via (docker root = project root):
+# docker build -t absaoss/eventgate:latest \
+# --build-arg TRUSTED_SSL_CERTS=./myTrustedCertsStorage \
+# --build-arg SASL_SSL_ARTIFACTS=./mySaslSslCredentials \
+# .
+#
+# run locally via:
+# docker run --platform=linux/amd64 -p 9000:8080 absaoss/eventgate:latest &
+# 
+# test via (provide payload):
+# curl http://localhost:9000/2015-03-31/functions/function/invocations -d "{payload}"
+#
+# Deploy to AWS Lambda via ACR
+
+FROM public.ecr.aws/lambda/python:3.11
+
+# Directory with TRUSTED certs in PEM format
+ARG TRUSTED_SSL_CERTS=./trusted_certs
+# Artifacts for kerberized sasl_ssl
+ARG SASL_SSL_ARTIFACTS=./sasl_ssl_artifacts
+
+# Import trusted certs before doing anything else
+RUN mkdir -p /opt/certs
+COPY $TRUSTED_SSL_CERTS /opt/certs/
+RUN for FILE in `ls /opt/certs/*.pem`; \
+do \
+    cat $FILE >> /etc/pki/tls/certs/ca-bundle.crt ;\
+done
+
+# Install
+#  -  basics
+RUN yum install -y wget tar xz bzip2-devel zlib-devel 
+#  -  GCC ("which" is required during some makefiles)
+RUN yum install -y which make gcc gcc-c++
+#  -  dependencies for kerberos SASL_SSL
+RUN yum install -y libffi-devel cyrus-sasl-devel cyrus-sasl-gssapi openssl-devel krb5-workstation
```

Comment:
This will create 3 layers, wouldn't it be better to combine them in one with `yum clean all` at the end?

Reply:
ultra-mega-super recombined but not discombobulated - looks better now?